docs/docs/recipes-deploying-your-site.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Deploying your site
```

Comment:
We should combine titles here so the same text isn't repeated in an h1 and h2 (and update them to follow the [Style Guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/#format-titles-and-headers)). Including the word "Recipes" will set these docs apart from other guides on similar topics, so that people will recognize the format from the top of the page. 

How about something like this?

```suggestion
title: Recipes: Deploying Your Site
```